### PR TITLE
[Fix] Unit42 Feed - *Feed Related Indicators* indicator field are not populated

### PR DESCRIPTION
**Pull request by bakatzir**
…cator field were not populated

## Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/26995
Not updating dockerimage due to: https://github.com/demisto/etc/issues/26353

## Description
The field is grid, but was not populated accordingly.

## Screenshots
indicators tab:
Screen Shot 2020-07-29 at 12 46 33
indicator layout:
Screen Shot 2020-07-29 at 12 54 40

## Minimum version of Demisto
- [x] 5.5.0

## Does it break backward compatibility?
   - [x] No

## Must have
- [x] Tests - only UT, no framework for integration tests.
- [x] Documentation

### CODE REVIEW

File: Packs/FeedUnit42/Integrations/FeedUnit42/FeedUnit42.py [hunk 1]
```diff
@@ -140,13 +140,15 @@ def parse_relationships(indicators: list, relationships: list = [], pivots: list
                             else:
                                 indicator['fields'][field_type] = [pivot_field]
                         else:  # a feedrelatedindicators is a list of dict
```
Comment (DeanArbel) [inline Packs/FeedUnit42/Integrations/FeedUnit42/FeedUnit42.py:142]: can you add a validation that feedrelatedindicators is a list of dict?

File: Packs/FeedUnit42/Integrations/FeedUnit42/FeedUnit42.py [hunk 2]
```diff
@@ -140,13 +140,15 @@ def parse_relationships(indicators: list, relationships: list = [], pivots: list
...
-                            external_ids = [item.get('external_id') for item in pivot_field if 'external_id' in item]
-                            indicator['fields'][field_type]['value'] = external_ids
-                            urls = [item.get('url') for item in pivot_field if 'url' in item]
-                            indicator['fields'][field_type]['description'] = urls
+                            indicator['fields'][field_type] = []
+                            for item in pivot_field:
+                                if 'url' in item or 'external_id' in item:
+                                    feedrelatedindicators_obj = {
+                                        'type': 'MITRE ATT&CK',
+                                        'value': item.get('external_id') if 'external_id' in item else None,
```
Comment (DeanArbel) [inline Packs/FeedUnit42/Integrations/FeedUnit42/FeedUnit42.py:148]: you can remove the ternary condition here and in the line below

File: Packs/FeedUnit42/ReleaseNotes/1_0_1.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Unit42 Feed
+- Fixed an issue where the *Feed Related Indicators* indicator field were not populated.
```
Comment (amshamah419) [inline Packs/FeedUnit42/ReleaseNotes/1_0_1.md:4]:
```suggestion
- Fixed an issue where the *Feed Related Indicators* indicator field was not populated.
```

**Comment by bakatzir** [comment]
fails on the dockerimage tag, mentioned in the relevant issue, force merging.
```
Validating Packs/FeedUnit42/Integrations/FeedUnit42/FeedUnit42.yml as integration
Validating scheme for Packs/FeedUnit42/Integrations/FeedUnit42/FeedUnit42.yml
Packs/FeedUnit42/Integrations/FeedUnit42/FeedUnit42.yml: [DO106] - The docker image tag is not the latest numeric tag, please update it.
The docker image tag in the yml file is: 1.0.0.9219
The latest docker image tag in docker hub is: 1.0.0.9904
You can check for the most updated version of demisto/taxii2 here: https://hub.docker.com/r/demisto/taxii2/tags
```

**Pull request merged**